src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -360,7 +360,7 @@ protected function shouldntReport(Throwable $e)
                 with($throttle->key ?: 'illuminate:foundation:exceptions:'.$e::class, fn ($key) => $this->hashThrottleKeys ? md5($key) : $key),
                 $throttle->maxAttempts,
                 fn () => true,
-                $throttle->decayMinutes
+                $throttle->decaySeconds
```

Comment:
This doesn't seem right, but it is. I have a fix up for this in the framework. Will rebase once it is merged.